webclient/core/repository_connector.py

```diff
@@ -697,7 +758,11 @@ def save_binary_file(
         )
         self._check_binary_file_container()
         url = self._get_request_url(FedoraRequestType.CREATE_BINARY_FILE)
-        result = self._send_request(url, FedoraRequestType.CREATE_BINARY_FILE)
+        headers = {
+            "Content-Type": "text/turtle",
+        }
+        rdf = f'<> <http://purl.org/dc/terms/creator> "{self.user}" .'
```

Comment:
Místo fixního užívání `http://purl.org/dc/terms/creator` bych definoval prefix `@prefix dcterms: <http://purl.org/dc/terms/> .` a pak referoval na `dcterms:creator`. Fedora s tím stejně takto pracuje, tak ať to nemate.